tests/e2e/lib/backup.go

```diff
@@ -77,3 +88,59 @@ func IsBackupCompletedSuccessfully(c *kubernetes.Clientset, ocClient client.Clie
 		GetVeleroContainerFailureLogs(c, backup.Namespace),
 	)
 }
+
+// https://github.com/vmware-tanzu/velero/blob/11bfe82342c9f54c63f40d3e97313ce763b446f2/pkg/cmd/cli/backup/describe.go#L77-L111
+func DescribeBackup(veleroClient veleroclientset.Interface, ocClient client.Client, namespace string, name string) (backupDescription string) {
```

Comment:
++

Reply:
It was just moved from
https://github.com/openshift/oadp-operator/pull/1463/files/e6b8141ff0860b5fb2b0356efe8c082d4ed40e8f#diff-ed46ecb3887b83856cf6a56eb504c902b156c6964915d00f8922c71879271dacL28